2_Clustering/1 Clustering Algorithms.ipynb

```diff
@@ -3869,38 +3869,109 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    plt.scatter(X_digits[:, 0], X_digits[:, 1], c=y_pred)
</pre><p>You should probably import matplotlib - </p><p>you can do</p><pre class="ql-syntax" spellcheck="false">from matplotlib import pyplot as plt
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WebValley2021ReImagined/unsupervised-learning/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>